spec/models/manageiq/providers/embedded_terraform/automation_manager/configuration_script_source_spec.rb

```diff
@@ -15,7 +15,21 @@
     let(:repos)              { Dir.glob(File.join(repo_dir, "*")) }
     let(:repo_dir_structure) { %w[hello_world.tf] }
 
+    def verify_req(req)
+      body = JSON.parse(req.body)
+      expect(body).to(have_key('templateZipFile'))
+    end
+
     before do
+      ENV["TERRAFORM_RUNNER_URL"] = "https://1.2.3.4:7000"
+      @hello_world_variables_response = JSON.parse(File.read(File.join(__dir__, "../../../../../lib/terraform/runner/data/responses/hello-world-variables-success.json")))
```

Comment:
I think you want `File.join(__dir__, "runner/data/responses/hello-world-variables-success.json")` here also

Reply:
Won't do,  different subdirectories in the parent directory tree,
Here only reusing already available data file from another spec

```
      Failure/Error: @hello_world_variables_response = JSON.parse(File.read(File.join(__dir__, "runner/data/responses/hello-world-variables-success.json")))
      
      Errno::ENOENT:
        No such file or directory @ rb_sysopen - /Users/manoj/git/ManageIQ/manageiq-providers-embedded_terraform/spec/models/manageiq/providers/embedded_terraform/automation_manager/runner/data/responses/hello-world-variables-success.json
      # ./spec/models/manageiq/providers/embedded_terraform/automation_manager/configuration_script_source_spec.rb:26:in `read'
```